api/envoy/config/overload/v3/overload.proto

```diff
@@ -80,6 +82,35 @@ message Trigger {
   }
 }
 
+// Typed configuration for the "envoy.overload_actions.reduce_timeouts" action.
+message ScaleTimersOverloadActionConfig {
+  enum TimerType {
+    UNSPECIFIED = 0;
```

Comment:
Is the reason this is specified to force the user to explicitly configure the timeout they want? I think that's OK but can you add a small comment?